app/src/main/java/com/example/yard/HelperClass/QueryPosts.java

```diff
@@ -0,0 +1,33 @@
+package com.example.yard.HelperClass;
+
+import com.example.yard.Adapter.PostCreation;
+import com.example.yard.Adapter.PostsAdapter;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+import com.parse.ParseUser;
+
+import java.util.List;
+
+public class QueryPosts {
+  public void queryPosts(boolean isFromBio, PostsAdapter adapter, List<PostCreation> allPosts) {
+    ParseQuery<PostCreation> query = ParseQuery.getQuery(PostCreation.class);
```

Comment:
This is added in another PR. https://github.com/Zaddyp/YARD/pull/15. Please remove it from this PR.